fixtures/eslint/lib/cli-engine/config-array-factory.js

```diff
@@ -578,7 +578,6 @@ class ConfigArrayFactory {
    * Normalize a given `.eslintignore` data to config array elements.
    * @param {string[]} ignorePatterns The patterns to ignore files.
    * @param {ConfigArrayFactoryLoadingContext} ctx The loading context.
-   * @returns {IterableIterator<ConfigArrayElement>} The normalized config.
```

Comment:
@calebeby Can you help me understand why this was removed? 🤔

Reply:
The JSDoc plugin was saying that this function had a @returns without a return statement. I think they should have been using @yields

Reply:
Does that mean this change should be switching `@returns` to `@yields` instead of deleting it? 🤔